ansible/roles/testbed/tasks/main.yml

```diff
@@ -48,12 +60,25 @@
     state: started
     start_mode: auto
 
-- name: Pull docker images
+- name: Set Windows Server version var
+  set_fact:
+    windows_server_version: "{{ ansible_distribution | regex_findall('Windows Server (\\d{4})') | first }}"
+
+- name: Pull docker images (Windows Server 2016)
   win_shell: "docker pull {{ item }}"
   with_items:
     - microsoft/nanoserver
     - microsoft/windowsservercore
-    - python:3.6.5
+    - python:3.7
```

Comment:
please move to role without `when` block

Reply:
Done